<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SendCoinsEntry</class>
 <widget class="QFrame" name="SendCoinsEntry">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>825</width>
    <height>150</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>825</width>
    <height>150</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>825</width>
    <height>150</height>
   </size>
  </property>
  <property name="lineWidth">
   <number>0</number>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frmEntry">
     <property name="minimumSize">
      <size>
       <width>808</width>
       <height>150</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>825</width>
       <height>150</height>
      </size>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="horizontalSpacing">
       <number>0</number>
      </property>
      <property name="verticalSpacing">
       <number>10</number>
      </property>
      <property name="margin">
       <number>20</number>
      </property>
      <item row="0" column="1">
       <widget class="QValidatedLineEdit" name="payTo">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="toolTip">
         <string>The address to send the payment to  (e.g. 4Zo1ga6xuKuQ7JV7M9rGDoxdbYwV5zgQJ5)</string>
        </property>
        <property name="maxLength">
         <number>34</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lblLabel">
        <property name="minimumSize">
         <size>
          <width>140</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>LABEL</string>
        </property>
        <property name="buddy">
         <cstring>addAsLabel</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QValidatedLineEdit" name="addAsLabel">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Enter a label for this address to add it to your address book</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="lblAmount">
        <property name="minimumSize">
         <size>
          <width>140</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>AMOUNT</string>
        </property>
        <property name="buddy">
         <cstring>payAmount</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0,0,0,0">
        <property name="spacing">
         <number>7</number>
        </property>
        <property name="sizeConstraint">
         <enum>QLayout::SetMaximumSize</enum>
        </property>
        <item>
         <widget class="BitcoinAmountField" name="payAmount">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="addressBookButton">
          <property name="minimumSize">
           <size>
            <width>70</width>
            <height>30</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Choose address from address book</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../bitcoin.qrc">
            <normaloff>:/icons/address-book</normaloff>:/icons/address-book</iconset>
          </property>
          <property name="shortcut">
           <string>Alt+A</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="pasteButton">
          <property name="minimumSize">
           <size>
            <width>70</width>
            <height>30</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Paste address from clipboard</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../bitcoin.qrc">
            <normaloff>:/icons/editpaste</normaloff>:/icons/editpaste</iconset>
          </property>
          <property name="shortcut">
           <string>Alt+P</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="deleteButton">
          <property name="minimumSize">
           <size>
            <width>70</width>
            <height>30</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Remove this recipient</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../bitcoin.qrc">
            <normaloff>:/icons/remove</normaloff>:/icons/remove</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="lblPayTo">
        <property name="minimumSize">
         <size>
          <width>140</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>PAY TO</string>
        </property>
        <property name="buddy">
         <cstring>payTo</cstring>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QValidatedLineEdit</class>
   <extends>QLineEdit</extends>
   <header>qvalidatedlineedit.h</header>
  </customwidget>
  <customwidget>
   <class>BitcoinAmountField</class>
   <extends>QLineEdit</extends>
   <header>bitcoinamountfield.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../bitcoin.qrc"/>
 </resources>
 <connections/>
</ui>
